packages/components/themes/dark.js

```diff
@@ -693,7 +693,8 @@ const Dark = {
       background: "#292929",
 
       border: "1px solid",
-      borderRadius: "0 3px 3px 0",
+      borderRadiusLtr: "0 3px 3px 0",
```

Comment:
It may be worth using the getCorrectFourValuesStyle utility

Reply:
Change to getCorrectBorderRadius utility